src/text_diff.rs

```diff
@@ -1,5 +1,3 @@
-use diff;
```

Comment:
Why remove this? Don't we need this to mark that we use the `diff` crate in the codebase?

And I want to use this comment to thank you for the change for clap's derive macro, it's really great :)

Reply:
Honestly I just removed it because clippy said to. But it still works because whenever we use things from the diff crate in this file, we’re referring to them with the prefix `diff::`. Runs fine and tests pass, but I may be missing something still.

Reply:
Maybe not, I didn't know you could directly use things like this.
`use` seems to be only necessary when you want to use things that are nested in a crate without having to mention the crate before every time, so all good, thanks (: